filtered-resource-service/src/main/java/uk/gov/gchq/palisade/service/filteredresource/config/ApplicationConfiguration.java

```diff
@@ -17,6 +17,7 @@
 package uk.gov.gchq.palisade.service.filteredresource.config;
 
 import akka.actor.typed.ActorRef;
+import akka.actor.typed.ActorSystem;
```

Comment:
I see here a shift to the Akka Typed.  Reads like a good idea, but feel that it should be an all or none.  Do we have a ticket to cover off the shift to Akka Typed for services?

Reply:
The typed approach is not mature enough to be used across the rest of the services but it is useful within the FRS.